### DISCUSSION TITLE
Is it possible to remove this leftover on Mail.com?

### DISCUSSION THREAD

**Discussion by Mishasama** [uBlockOrigin/uAssets#24971] [Q&A]
![image](https://github.com/user-attachments/assets/5f19d7fa-0249-4bca-bb82-6af71c3eb890)

Although I have hidden this div element, it still leaves an empty space due to the following parameter. 

![image](https://github.com/user-attachments/assets/3b394089-2ffa-43ed-84d8-b269f8ac2e3a)

Is there a way to set it to 0?

**Comment by garry-ut99** [comment]
> Is there a way to set it to 0?


https://github.com/gorhill/uBlock/wiki/Static-filter-syntax#subjectstylearg

I don't have an account there, but from your screenshoot, try one of the following:
`mail.com##div.section.section-content.section-1:style(margin-right: 0px !important;)` or
`mail.com##div.section.section-content.section-1[style]:style(margin-right: 0px !important;)`

**Reply by Mishasama** [reply]
Amazing! Both works great! Thank you very much!
